device_test.go

```diff
@@ -156,6 +175,36 @@ func TestClaimAndRelease(t *testing.T) {
 	}
 }
 
+func TestInterfaceDescriptionError(t *testing.T) {
+	_, done := newFakeLibusb()
+	defer done()
+
+	for _, tc := range []struct {
+		name           string
+		cfg, intf, alt int
+	}{
+		{"no config", 2, 1, 1},
+		{"no interface", 1, 3, 1},
+		{"no alt setting", 1, 1, 5},
+	} {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
Can this sub-test be marked t.Parallel()? If not, add a comment explaining why not.

Reply:
Done.

Reply:
(after looking at it, it didn't work parallelized - t.Parallel deferred the execution of the test until such a point where libusb was already restored to a real implementation.
I can think of a way to improve the fakelibusb and NewContext() to avoid using a shared global state in the tests. But in a later PR.